apps/catalogue/src/views/ResourceDetailsView.vue

```diff
@@ -155,118 +63,49 @@ export default {
     };
   },
   computed: {
-    row() {
+    resourceData() {
       return this.data[0];
     },
-  },
-  methods: {
-    openRouterLink(event, name) {
-      this.$router.push({
-        name: name + "-details",
-        params: event[Object.keys(event)[0]],
-      });
-    },
-    toggleNA() {
-      this.hideNA = !this.hideNA;
-    },
-    toSentence(str) {
-      return (
-        str
-          // insert a space before all caps
-          .replace(/([A-Z])/g, " $1")
-          // uppercase the first character
-          .replace(/^./, function (str) {
-            return str.toUpperCase();
-          })
-      );
-    },
-    renderValue(row, col) {
-      if (row[col.name] === undefined) {
-        return [];
-      }
-      if (
-        col.columnType == "REF_ARRAY" ||
-        col.columnType == "MREF" ||
-        col.columnType == "REFBACK" ||
-        col.columnType == "ONTOLOGY_ARRAY"
-      ) {
-        return row[col.name].map((v) => {
-          if (col.refLabel) {
-            return this.applyJsTemplate(col.refLabel, v);
+    sections() {
```

Comment:
nice idea, now you can hide complete sections in case all items are NA?

Reply:
yes  , i would suggest doing that , did not add it to the PR to keep original behaviour , but this setup allow to easily implement doing things to entire sections.